superset/security/manager.py

```diff
@@ -656,6 +657,11 @@ def _get_pvms_from_builtin_role(self, role_name: str) -> List[PermissionView]:
                         role_from_permissions.append(pvm)
         return role_from_permissions
 
+    def find_role_by_id(self, role_id: int) -> Any:
+        return (
+            self.get_session.query(self.role_model).filter_by(id=role_id).one_or_none()
+        )
```

Comment:
As we're actually doing a for-loop when retrieving these, to avoid unnecessary query overhead it might be better to do `find_roles_by_id(self, role_ids: List[int]) -> List[Role]`. Check https://github.com/apache/superset/blob/77093a874e8c4c81c6e3b8557bca545c3fc0f3a2/superset/dao/base.py#L49-L77 for the DAO implementation